src/components/DateRangePicker.js

```diff
@@ -448,6 +482,7 @@ const DateRangePicker = React.createClass({
         position: 'absolute',
         left: this.props.isRelative ? 'auto' : 0,
         right: 0,
+        width: window.innerWidth < 450 && '100%',
```

Comment:
Will this throw a css error or warning by assigning width to false in the case that it's greater than 450?